tasks.py

```diff
@@ -577,7 +577,7 @@ def ensure_paths(c):
 @task(ensure_paths)
 def start_docker(c):
     """Docker entry point task. Do not call!"""
-
+    c.run("python -m flask create-db", echo=True)
```

Comment:
In der story wurde von Create and Migrate gesprochen.
Deswegen solltest du hier glaube ich einfach create-and-load db verwenden.

Reply:
Diese action würde bei jedem Start die Datenbank neu mit testdaten füllen. Das darf aber im Produktivbetrieb nicht passieren,

Reply:
Dann vielleicht nochmal mit Daniel abklären.
Vielleicht ist ein "Wenn datenbank leer, dann Testdaten einfügen" Befehl dann sinvoll.